presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -73,12 +72,8 @@
     private int limitLargeForSegment = DEFAULT_LIMIT_LARGE_FOR_SEGMENT;
     private int topNLarge = DEFAULT_TOPN_LARGE;
 
-    private Duration idleTimeout = DEFAULT_IDLE_TIMEOUT;
```

Comment:
Many other consts in this file will not be used because of the deletion. Can we remove them? (e.g., DEFAULT_IDLE_TIMEOUT , DEFAULT_MAX_BACKLOG_PER_SERVER, etc)